api/subscriptions/serializers.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import serializers as ser
+from rest_framework.exceptions import ValidationError
+
+from api.base.serializers import JSONAPISerializer
+
+NOTIFICATION_TYPES = {
+    'none': 'none',
+    'instant': 'email_transactional',
+    'daily': 'email_digest'
+}
+
+
+class FrequencyField(ser.Field):
+    def to_representation(self, obj):
+        user = self.context['request'].user
```

Comment:
Same as above, pseudocode:
```python
user_id = user = self.context['request'].user.id # if user obj, I don't know what it is here
if obj.none.filter(id=user_id).exists():
   return 'none'
if obj.email_transactional.filter(id=user_id).exists():
   return 'instant'
return 'daily'
```
The last check may be desirable (and throw an error if not found) but right now frequency could be undefined and cause an error.